poonam/JS/Assignment20_js/que2.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Que 2</title>
+        <style>
+            table,
+            td,
+            tr,
+            th {
+                border: 1px solid black;
+                border-collapse: collapse;
+                width: 300px;
+                text-align: center;
+            }
+        </style>
+    </head>
+    <body>
+        <table id="tdata">
+            <tr>
+                <td>Sr. No.</td>
+                <td>Name</td>
+            </tr>
+            <tr id="row1">
+                <td>1.</td>
+                <td>Poonam</td>
+            </tr>
+            <tr id="row2">
+                <td>2.</td>
+                <td>Neha</td>
+            </tr>
+        </table>
+        <br>
+        Enter no of row: <input type="text" id="rowno"><br><br>
+        Enter no of column: <input type="text" id="colno"><br><br>
+        Enter data to update: <input type="text" id="data"><br><br>
+        <button onclick="add()">Add row</button>
+        <button onclick="changeContent()">Update</button>
+        <script>
+            function add() {
```

Comment:
This add row Function not working properly.
```suggestion
            function add() {
```

Reply:
it is working on my side

Reply:
But it's added only empty row not value with row.

Reply:
i added changeContent function for adding data